source/CIGameScene.cpp

```diff
@@ -325,16 +344,18 @@ void GameScene::processSpecialStardust(const cugl::Size bounds, const std::share
                 stardustQueue->addStardust(stardust->getColor(), bounds);
                 stardustQueue->addStardust(stardust->getColor(), bounds);
                 stardustQueue->addStardust(stardust->getColor(), bounds);
-                stardustQueue->addStardust(CIColor::getRandomColor(), bounds);
-                stardustQueue->addStardust(CIColor::getRandomColor(), bounds);
+                stardustQueue->addStardust(CIColor::getRandomColor(_colorCount), bounds);
+                stardustQueue->addStardust(CIColor::getRandomColor(_colorCount), bounds);
                 break;
             case StardustModel::Type::SHOOTING_STAR:
                 CULog("SHOOTING STAR");
                 stardustQueue->addShootingStardust(stardust->getColor(), bounds);
                 stardustQueue->addShootingStardust(stardust->getColor(), bounds);
+                [[fallthrough]];
```

Comment:
you can get rid of the [[failthrough]] here. I put breaks in the pr I just put up